sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4761,7 +4761,7 @@ pi_result piEventCreate(pi_context Context, pi_event *RetEvent) {
   if (ZeAllHostVisibleEvents) {
     ZeEventDesc.signal = ZE_EVENT_SCOPE_FLAG_HOST;
   } else {
-    ZeEventDesc.signal = ZE_EVENT_SCOPE_FLAG_DEVICE;
```

Comment:
I don't completely understand this change. So do you basically want to change this to ZE_EVENT_SCOPE_FLAG_SUBDEVICE?

Reply:
No, the current recommendation from L0 GPU RT is that "0" scope should be used. The ZE_EVENT_SCOPE_FLAG_SUBDEVICE is "1".

Reply:
Hm, but per specification it looks like ZE_EVENT_SCOPE_FLAG_SUBDEVICE is "0" and ZE_EVENT_SCOPE_FLAG_DEVICE is "1".
https://spec.oneapi.io/level-zero/latest/core/api.html?highlight=ze_event_scope_flag_device#_CPPv426ZE_EVENT_SCOPE_FLAG_DEVICE

Reply:
ZE_BIT(0) = 1 << 0 = 1

Reply:
Ok, sorry